packages/vite/src/node/plugins/reporter.ts

```diff
@@ -242,6 +247,12 @@ export function buildReporterPlugin(config: ResolvedConfig): Plugin {
         )
       }
     },
+
+    closeBundle() {
+      if (shouldLogInfo && !config.build.watch) {
+        config.logger.info(`built in ${displayTime(+new Date() - startTime)}`)
```

Comment:
Currently this looks like:

```
vite v4.1.2 building for production...
✓ 3 modules transformed.
dist/index.html                0.12 kB
dist/assets/index-884e2e9e.js  1.57 kB │ gzip: 0.59 kB
built in 59ms
```

The last line doesn't quite stand out. Maybe this could do:

```suggestion
        config.logger.info(`${colors.green(`✓`)} built in ${displayTime(+new Date() - startTime)}`)
```

Maybe the time could use a bold, but this can be improved later too.